Exiled.API/Features/Roles/Scp049Role.cs

```diff
@@ -137,7 +137,7 @@ public float CallCooldown
         }
 
         /// <summary>
-        /// Gets or sets the amount of time before SCP-049 can use its Good Sense ability again.
+        /// Gets or sets the amount of time before SCP-049 can use its Good Sense of the Doctor ability again.
```

Comment:
I can be incorrect, but I think previous version was more correct

Reply:
The full ability's name is The Good Sense of the Doctor, while it can be it's "good sense" I think its important to be clearer as a whole